.github/workflows/ci.yml

```diff
@@ -42,3 +42,12 @@ jobs:
           path: |
             **/build/reports/tests/
             **/out/failures/
+
+  generate-dokka:
+    name: Generate Dokka
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/stripe_setup
+      - name: Generate Dokka Documentation
+        run: ./gradlew dokkaHtml
```

Comment:
We do `./gradlew dokkaHtmlMultiModule` as part of release. I think we should match that here.

Reply:
It takes a very long time to run this command, which is why I opted for single module